pkg/loop/internal/relayer/pluginprovider/ext/median/test/datasource.go

```diff
@@ -52,7 +59,14 @@ func (s staticDataSource) Evaluate(ctx context.Context, ds median.DataSource) er
 		return fmt.Errorf("failed to observe dataSource: %w", err)
 	}
 	if gotVal.Cmp(s.Value) != 0 {
-		return fmt.Errorf("expected Value %s but got %s", value, gotVal)
+		return fmt.Errorf("expected Value %s but got %s", s.Value, gotVal)
```

Comment:
this was a prior typo where it was using the global variable value instead of the static data source's value